### PR TITLE
KFSPTS-14448 upgrade to the 1/24/2019 version of financials and get t…

### PR DESCRIPTION
**Pull request by db483**
…hings to compile

### CODE REVIEW

File: src/main/java/org/kuali/kfs/sys/batch/BatchFileUtils.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.core.api.util.KeyValue;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BatchFileUtils {
```
[the remark this answers is not in the document]
Comment (db483) [reply src/main/java/org/kuali/kfs/sys/batch/BatchFileUtils.java]: You are right, I will remove the file.

**Comment by db483** [comment]
I have updated the invoiceGeneral.tag file.

**Comment by db483** [comment]
Added Andrew as a reviewer as well to confirm I have captured all the changes that were needed for KFSPTS-13258 in the updated invoiceGeneral.tag.

**Comment by cah292** [comment]
FYI, the KFSPTS-13258 changes were reverted again today. If the next set of KFSPTS-13258 revisions get committed in time, then it may still be possible to merge this PR as-is; otherwise, the invoiceGeneral.tag overlay will need to be removed from this PR.

**Comment by db483** [comment]
Thank you for the note Chad. I will re-base and add needed changes on Friday before you merge.

**Comment by db483** [comment]
Chad, I have rebased this branch and made the needed changes after Andrew's revert. This is now ready for review again.

**Pull request merged**